mailer/index.js

```diff
@@ -59,9 +59,13 @@ const EMAIL_CONTENT = {
   },
   RAISE_CONCERN: {
     subject: 'Concern Raised',
-    contentTop: `A concern has been raised on your offer to [User name]. The question states [question]. `,
+    contentTop: `A concern has been raised on your offer to [User name]. The question states [question].`,
     contentBottom: `Check your dashboard to resolve this concern.`,
   },
+  RESOLVE_CONCERN: {
+    subject: 'Concern Resolved',
+    contentTop: `Your raised concern has been resolved. Check your dashboard for more information.`,
```

Comment:
Not really bothered about the implementation here, but I think it would make more sense if the question and the response is given in the mail so that the user can read them without logging into the dashboard.

```suggestion
contentTop: `Your raised concern has been resolved. <br /> <strong>[Question]</strong> <br /> <br /> [Response]`,
```